packages/hardhat-core/test/fixture-projects/compilation-import-folder/contracts/A.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.19;
+
+// The file has to not exist for test purposes
+import "some-plugin"
+
+contract A {}
```

Comment:
```suggestion
contract A {}

```

Reply:
Our `.editorconfig` file has `insert_final_newline = true`. @ChristopherDedominici can you double-check that your IDE is using the `.editorconfig` settings? Maybe you need to install [this](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig).